compaction/compaction_manager.cc

```diff
@@ -1582,9 +1582,6 @@ class rewrite_sstables_compaction_task_executor : public sstables_task_executor
 
             setup_new_compaction(descriptor.run_identifier);
 
-            compaction_backlog_tracker user_initiated(std::make_unique<user_initiated_backlog_tracker>(_cm._compaction_controller.backlog_of_shares(200), _cm.available_memory()));
-            _cm.register_backlog_tracker(user_initiated);
-
```

Comment:
There's another one in run_cleanup_job().

Should we direct it to the maintenance group too? I think we should.

Reply:
I think it makes sense so cleanup won't interfere with regular compaction (I am particularly worried about cleanup stealing resources from regular compaction that would be devoted to compacting new (L0) sstables, to avoid a scenario where sstables are produced faster than we compact them, which causes memory issue on read path), and it's part of maintenance anyway (post topology ops). I think it's a positive change.

Reply:
New PR at https://github.com/scylladb/scylladb/pull/20582